app/models/post.rb

```diff
@@ -0,0 +1,32 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def most_recent_comments
+    comments.order('created_at DESC').limit(5)
+  end
+end
+
+# class Post < ApplicationRecord
+#   belongs_to :author, class_name: 'User'
+#   has_many :comments
+#   has_many :likes
+
+#   after_save :update_post_counter
+
+#   def update_post_counter
+#     author.increment!(:posts_counter)
+#   end
+
+#   def five_recent_comments
+#     # comments.order(created_at: :desc).limit(5)
+#     comments.order('created_at DESC').limit(5)
+#   end
+# end
```

Comment:
### [ OPTIONAL]

- [ ] It is a great practice to always delete commented code in your application. This ensures that your code is clean. Remember you can always use git history to go back to the previous version of your code.